### PR TITLE
Enable autoline for RWA007-A

### PR DESCRIPTION
**Pull request by dbkbali**
# Description

# Contribution Checklist

- [ ] PR title starts with `(PE-<TICKET_NUMBER>)`
- [ ] Code approved
- [ ] Tests approved
- [ ] CI Tests pass

# Checklist

- [ ] Every contract variable/method declared as public/external private/internal
- [ ] Consider if this PR needs the `officeHours` modifier override
- [ ] Verify expiration (`30 days` unless otherwise specified)
- [ ] Verify hash in the description matches [here](https://emn178.github.io/online-tools/keccak_256.html)
- [ ] Validate all addresses used are in Goerli changelog or known
- [ ] Notify any external teams affected by the spell so they have the opportunity to review
- [ ] Deploy spell to Goerli `ETH_GAS="XXX" ETH_GAS_PRICE="YYY" make deploy`
- [ ] Ensure contract is verified on `Goerli` etherscan
- [ ] Change test to use Goerli spell address and deploy timestamp
- [ ] Cast spell on Goerli `make spell="0x-deployed-spell-address" cast-spell`
- [ ] Run `make archive-spell` or `make date="YYYY-MM-DD" archive-spell` to make an archive directory and copy `Goerli-DssSpell.sol`, `Goerli-DssSpell.t.sol`, `Goerli-DssSpell.t.base.sol`, and `Goerli-DssSpellCollateralOnboarding.sol`
- [ ] `squash and merge` this PR

### CODE REVIEW

File: src/Goerli-DssSpell.sol [hunk 1]
```diff
@@ -52,8 +52,7 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     function actions() public override {
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
-        // onboardCollaterals();
-        // offboardCollaterals();
```
Comment (iamchrissmith) [inline src/Goerli-DssSpell.sol:56]: I would leave these in, just commented out

File: src/Goerli-DssSpellCollateral.sol [hunk 2]
```diff
@@ -32,135 +32,31 @@ contract DssSpellCollateralAction {
...
-        //     _what:      "mat",
-        //     _startTime: block.timestamp,
-        //     _start:     CURRENT_XXX_A_MAT,
-        //     _end:       TARGET_XXX_A_MAT,
-        //     _duration:  30 days
-        // });
+    function updateCollaterals() internal {
+        // ------------------------------- Collateral updates -------------------------------
 
+        // Enable autoline for MIP65
```
Comment (iamchrissmith) [inline src/Goerli-DssSpellCollateral.sol:54]: I would include the forum link 
```suggestion
        // Enable autoline for MIP65
        // https://forum.makerdao.com/t/rwa007-mip65-monetalis-clydesdale-ces-domain-team-assessment/17787
```

File: src/Goerli-DssSpell.sol [hunk 3]
```diff
@@ -52,8 +52,7 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
...
+        updateCollaterals();
```
Comment (iamchrissmith) [inline src/Goerli-DssSpell.sol:55]:
```suggestion
        // onboardCollaterals();
        updateCollaterals();
        // offboardCollaterals();
```

**Review by iamchrissmith — CHANGES_REQUESTED** [review]
couple of minor comments

**Comment by iamchrissmith** [comment]
tests pass

```rust
Running 1 test for src/test/starknet.t.sol:StarknetTests
[PASS] testStarknet() (gas: 682022)
Test result: ok. 1 passed; 0 failed; finished in 434.62ms

Running 14 tests for src/Goerli-DssSpell.t.sol:DssSpellTest
[PASS] testCastCost() (gas: 455266)
[PASS] testFailTooEarly() (gas: 12594)
[PASS] testFailTooLate() (gas: 12549)
[PASS] testFailWrongDay() (gas: 12594)
[PASS] testFail_notScheduled() (gas: 14161)
[PASS] testOnTime() (gas: 451699)
[PASS] testSpellIsCast_GENERAL() (gas: 22696364)
[PASS] test_auth() (gas: 20963551)
[PASS] test_auth_in_sources() (gas: 4899216)
[PASS] test_bytecode_matches() (gas: 1047689)
[PASS] test_chainlog_values() (gas: 5758055)
[PASS] test_chainlog_version_bump() (gas: 2996306)
[PASS] test_nextCastTime() (gas: 351611)
[PASS] test_use_eta() (gas: 350466)
Test result: ok. 14 passed; 0 failed; finished in 3.71s
```

**Review by godsflaw — COMMENTED** [review]
looks good with @iamchrissmith's suggestions made.

**Review by iamchrissmith — APPROVED** [review]
changes look good. i'll merge this into the base branch for this week. thanks!

**Pull request merged**